src/VisualStudio/Core/Def/Implementation/Diagnostics/VisualStudioDiagnosticAnalyzerExecutor.cs

```diff
@@ -151,6 +151,11 @@ private CompilationWithAnalyzers CreateAnalyzerDriver(CompilationWithAnalyzers a
 
         private void ReportAnalyzerExceptions(Project project, ImmutableDictionary<DiagnosticAnalyzer, ImmutableArray<DiagnosticData>> exceptions)
         {
+            if (exceptions == null)
```

Comment:
@jasonmalinowski I dont believe that is true. InvokeAsync being able to return default(T) is contract. and all user has to deal with it. same as a method that can return null, caller need to guard for it.

if you mean existing caller can have such issue, I went thorough all officially supported features and made sure they are okay.